webapp/accounts/forms.py

```diff
@@ -98,6 +100,7 @@ def save(self, commit=True):
             user = super(VokoUserFinishForm, self).save(commit=False)
             user.set_password(self.cleaned_data["password1"])
             user.is_active = True
+            user.activated = datetime.now(UTC)
```

Comment:
I think the preferred way nowadays is
```
from django.utils import timezone

now = timezone.now()
```

Reply:
Done. It did seem a bit weird to use `timezone.now()` just for this one, but then again I see the call in the migration files and we have `USE_TZ = True`.